src/styles/variables.scss

```
@@ -0,0 +1,4 @@
+$font: Arial, Helvetica, sans-serif;
+$days: ("sun": 606px, "mon": 0, "tue": 101px, "wen": 202px, "thu": 303px , "fri": 404px, "sat": 505px);
```

```suggestion
$days: ("mon": 0, "tue": 1, "wen": 2, "thu": 3 , "fri": 4, "sat": 5, "sun": 6);
```
it will make your app much more responsive